### DISCUSSION TITLE
tape dimensions

### DISCUSSION THREAD

**Discussion by 55QA67** [frc5567/2023-Robot#153] [Team Posts]
I don't have access to the vision team so I'm posting this here. Here are the dimensions for the tape on the rocket. We should be able to measure over the correct amount and we're good. 
![screenshot 54](https://user-images.githubusercontent.com/25637778/52658882-20cba800-2eca-11e9-87ba-768249960c9c.png)

**Comment by coverbeek** [comment]
Everyone should have access to all of the sub-teams. You just don't get notifications if you're not a member... Thanks for posting this, nonetheless.

**Comment by 55QA67** [comment]
@coverbeek how's that for some googlefu?

**Comment by 55QA67** [comment]
I can't see the vision team. dik why, but I can't.

**Comment by MikasaSucasa** [comment]
You see this
![image](https://user-images.githubusercontent.com/7189454/52659237-dd256e00-2eca-11e9-83a9-01c1619b0c17.png)

**Comment by MikasaSucasa** [comment]
Click on the word "Teams"

**Comment by 55QA67** [comment]
yeah. and I just figured out how to see the vision team, so nvm.

**Comment by MikasaSucasa** [comment]
We will need the dimensions for the hole being covered since we should not trust the accuracy of their cuts
